tests/ha/test_ha_pod_restart.py

```diff
@@ -2929,14 +2929,15 @@ def test_continuous_reads_during_pod_restart_ros(self):
             "Step 6: Perform READs and verify DI on the written data in background during "
             "pod restart using %s method", self.restore_method)
         args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
-                'nclients': 1, 'nsamples': 5, 'skipwrite': True, 'skipcleanup': True,
+                'nclients': 1, 'nsamples': 20, 'skipwrite': True, 'skipcleanup': True,
                 'output': output}
 
         thread = threading.Thread(target=self.ha_obj.event_s3_operation,
                                   args=(event,), kwargs=args)
         thread.daemon = True  # Daemonize thread
         thread.start()
-        time.sleep(HA_CFG["common_params"]["20sec_delay"])
+        # TODO Need to update timing once we get stability in degraded IOs performance
+        time.sleep(200)
```

Comment:
Why delay in increased, please don't use hardcoded values.

Reply:
> Why delay in increased, please don't use hardcoded values.

As added in TODO, cluster is degradded state, so IOs are taking more time or might be getting stuck.  updated same https://jts.seagate.com/browse/CORTX-29928. Hence to make sure s3bench read/validate running parallel when pod is re-starting, increased the delay. Updating the hardcoded value.